packages.config

```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you add this back in for now? There's some associated cleanup that should be done with its removal.